cloudwatch-logs-to-logstash/config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Worth naming this file with `-template` or something, to make sure we don't commit accidentally?